React/Search input/src/components/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+const ENTER_KEY = "Enter";
+
+const ImmediateInput = (props) => {
+  const { placeholder, onSearch } = props;
+
+  const handleChange = (e) => {
+    onSearch(e.target.value);
+  };
+
+  return <input placeholder={placeholder} onChange={handleChange} />;
+};
```

Comment:
<input placeholder={placeholder} is duplicated in each component. Try to avoid dublication